src/build_workflow/opensearch_dashboards/build_artifact_check_plugin.py

```diff
@@ -17,21 +17,25 @@ class BuildArtifactOpenSearchDashboardsCheckPlugin(BuildArtifactCheck):
     def check(self, path):
         if os.path.splitext(path)[1] != ".zip":
             raise BuildArtifactCheck.BuildArtifactInvalidError(path, "Not a zip file.")
-        opensearch_dashboards_version = re.sub(r"-SNAPSHOT$", "", self.target.opensearch_version)
-        match = re.search(rf"^(\w+)-{opensearch_dashboards_version}.zip$", os.path.basename(path))
+
+        match = re.search(r"^(\w+)-[\d\.]*.*.zip$", os.path.basename(path))
```

Comment:
Will single digit be a problem?
We used to have things like 1.13.0 in ODFE, which is 2 digits after all.

Reply:
It shouldn't. This regex just says any combination of numbers and dots.

Reply:
Thanks, just confirmed by running it through a tester.